app/assets/stylesheets/application.scss

```diff
@@ -66,6 +73,24 @@
    //  @include vertical-align;
  }
 
+ .ghost-button-thick-border {
+   display: inline-block;
```

Comment:
Properties should be ordered background-color, border, color, display, font-weight, outline, padding, text-align, text-decoration, width